### PR TITLE
Feature to refund an item

### PR DESCRIPTION
**Pull request by u-blavins**
- Refund an item
- Refund a broken item and assign staff department to cost
- Error messaging

### CODE REVIEW

File: static/returnItems.js [hunk 1]
```diff
@@ -1,4 +1,144 @@
...
+                'Box(es)': 'Box',
+                'Millilitre(s)': 'ml',
+                'Litre(s)': 'Litre',
+                'Gram(s)': 'Gram',
+                'Kilogram(s)': 'Kilogram'
+            }
+            var sel = document.createElement('select');
+            sel.setAttribute('name', 'unitTypes[]');
+            sel.setAttribute('class', 'dropdownList tableInput');
+            ele.required = true;
```
Comment (LydiaLane) [inline static/returnItems.js]: Change 'ele' to 'sel'

File: static/returnItems.js [hunk 2]
```diff
@@ -1,4 +1,144 @@
...
+        if (c == 3 ) { // quantity
+            var options = {
+                'Return Option': '',
+                'Refund': 'refund',
+                'Broken': 'broken'
+            }
+            var sel = document.createElement('select');
+            sel.setAttribute('name', 'returnOption[]');
+            sel.setAttribute('class', 'dropdownList tableInput');
+            ele.required = true;
```
Comment (LydiaLane) [inline static/returnItems.js]: Change 'ele' to 'sel'

**Review by LydiaLane — CHANGES_REQUESTED** [review]

File: static/returnItems.js (hunk 2 above)
Comment (u-blavins) [reply static/returnItems.js]: Good spot will get that changed.

**Review by u-blavins — COMMENTED** [review]
That has been done

**Pull request merged**